packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -57,70 +57,6 @@ void main() {
     });
   }, skip: true); // [intended] API testing
 
-  group('findsOneWidget', () {
```

Comment:
To review the changes to these tests, take a look at [this commit](https://github.com/flutter/flutter/pull/127137/commits/b41b78bdb5637229d3d6f90155051d410fe826fa) which has just the changes and not the move to the `matchers_test.dart` file.